packages/api/src/form/ro/form.ro.ts

```diff
@@ -0,0 +1,7 @@
+import { FormPayload } from '@ehpr/common/form-payload';
+
+export class FormRO {
+  id!: number;
```

Comment:
Is this the short, readable reference ID that we'll supply to the user?

Reply:
I foresee this as being something we would use in a portal. I don't see it as something the user would interact with.